common-ts/src/utils/math.ts

```diff
@@ -0,0 +1,74 @@
+import { BN, L2OrderBook, PERCENTAGE_PRECISION } from '@drift-labs/sdk';
+
+const calculatemarkPrice = (bestBidPrice: BN, bestAskPrice: BN) => {
```

Comment:
nitpick but this and the function below should be Mark to keep the camelcase consistent